File: Iniciante/URI 2786.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
    
var L = parseInt(lines.shift());
var C = parseInt(lines.shift());
var lajes1 = (L * C) + ((L - 1) * (C - 1));
var lajes2 = ((L - 1) * 2) + ((C - 1) * 2);  
 
console.log(lajes1);
console.log(lajes2);

File: Iniciante/URI 2786.r
input = file('stdin', 'r')

l = as.integer(readLines(input, n=1))
c = as.integer(readLines(input, n=1))

lajes1 = (l * c) + ((l - 1) * (c - 1))
lajes2 = ((l - 1) * 2) + ((c - 1) * 2)

write(lajes1, '')
write(lajes2, '')